iiod/iiod.c

```diff
@@ -481,7 +534,7 @@ static int main_server(struct iio_context *ctx, bool debug)
 	}
 
 	if (ipv6)
-		IIO_INFO("IPv6 support enabled\n");
+		IIO_ERROR("IPv6 support enabled\n");
```

Comment:
Why is this an error?

Reply:
This was an accident - I was easier to change things to IIO_ERROR that recompile the entire library, and this never got changed back. Thanks for looking.